api/urls.py

```diff
@@ -148,6 +149,7 @@
         ActionableCanteenRetrieveView.as_view(),
         name="retrieve_actionable_canteen",
     ),
+    path("siret/<str:siret>", SiretCheckView.as_view(), name="siret_check"),
```

Comment:
On pourrait parler en live sur le path de cet endpoint ?

1. D'un coté ça peut être nommé comme un endpoint RPC et donc inclure un verbe dans le nom, à l'image des endpoints `importPurchases`, `addManager`, etc. Si c'est le cas, un POST sur un endpoint style `verifySiretStatus` ou `checkSiretStatus` serait préférable.

2. D'un autre coté on peut dire qu'on obtient des infos sur la ressource `canteen` et donc c'est un endpoint REST. Mais la ressource n'est pas le siret en soit, donc un endpoint `/siret/<siret>` n'est pas descriptive. Généralement dans ces cas on privilégie `/canteen/siret/<str:siret>`.

J'ai une légère préférence pour la première option car on n'est pas en train de retourner les infos sur la cantine elle même, mais plutôt les infos nécessaires seulement à la gestion de l'étape de remplissage du SIRET, mais partant pour en discuter.

Reply:
Qqch à garder en tête avec le nom, quand on ajoute l'appel vers l'API externe, on pourrait donner un réponse avec le titre et localisation etc de la cantine. On peut en discuter en live quand tu veux

Reply:
/canteenStatus/siret/<str:siret>
(maybe add a serializer)

Reply:
Added serializer and renamed stuff here : https://github.com/betagouv/ma-cantine/pull/2202/commits/b30641254309678170ae84d058801ecd6c7df87f